src/index.tsx

```diff
@@ -216,1753 +216,5 @@ export type {Locale} from './utils/locale';
 export type {TrackingEvent, IconProps, DataAttributes} from './utils/types';
 export type {RegionCode} from './utils/region-code';
 
-/*
- * Temporary solution to export Community components until Mistica gets migrated to ESModules
- * the community.js export has issues because it exports an ES module and next12 interterprets it as a CommonJS module
- * importing from /dist/ is not an option because those modules don't get the context from the theme provider
- */
-export {default as CommunityExampleComponent} from './community/example-component';
-export {default as CommunityAdvancedDataCard} from './community/advanced-data-card';
-export {
-    RowBlock as CommunityRowBlock,
-    SimpleBlock as CommunitySimpleBlock,
-    InformationBlock as CommunityInformationBlock,
-    HighlightedValueBlock as CommunityHighlightedValueBlock,
-    ValueBlock as CommunityValueBlock,
-    ProgressBlock as CommunityProgressBlock,
-} from './community/blocks';
-export {default as iconKeywords} from './generated/mistica-icons/icons-keywords';
-
-// Exported this way to facilitate tree-shaking
```

Comment:
no more copy paste needed. The icons are exported as `export * from './generated/mistica-icons';` (see the end of this file)